backend/src/middleware/handleHtmlContent/handleContentData.js

```diff
@@ -0,0 +1,69 @@
+import extractMentionedUsers from './notifications/extractMentionedUsers'
+import extractHashtags from './hashtags/extractHashtags'
```

Comment:
Hashtags are in the folder `hashtags` not sure this is appropriate but its certainly better then in notifications

Reply:
I find the locations kind of clear. @ulfgebhardt